FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -234,20 +239,21 @@ - (void)refreshInstanceIDTokenAndFetchCheckInInfoWithCompletionHandler:
                                                     }]];
     }
 
-    // If the token is available, try to get the instanceID.
+    // We have a valid token. Get the backing installationID.
     __weak RCNConfigFetch *weakSelf = self;
-    [instanceID getIDWithHandler:^(NSString *_Nullable identity, NSError *_Nullable error) {
+    [installations installationIDWithCompletion:^(NSString *_Nullable identifier,
+                                                  NSError *_Nullable error) {
       RCNConfigFetch *strongSelf = weakSelf;
 
       // Dispatch to the RC serial queue to update settings on the queue.
       dispatch_async(strongSelf->_lockQueue, ^{
         RCNConfigFetch *strongSelfQueue = weakSelf;
 
         // Update config settings with the IID and token.
-        strongSelfQueue->_settings.configInstanceIDToken = [token copy];
-        strongSelfQueue->_settings.configInstanceID = identity;
+        strongSelfQueue->_settings.configInstallationsToken = [tokenResult.authToken copy];
```

Comment:
Nit: `copy` is not required here because `configInstallationsToken` property is defined with `copy` attribute already.

Reply:
Done.